src/legacy/server/i18n/localization/telemetry_localization_collector.ts

```diff
@@ -19,25 +19,36 @@
 
 import { i18nLoader } from '@kbn/i18n';
 import { size } from 'lodash';
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
 import { getIntegrityHashes, Integrities } from './file_integrity';
-import { KIBANA_LOCALIZATION_STATS_TYPE } from '../../../common/constants';
-import { UsageCollectionSetup } from '../../../../../../plugins/usage_collection/server';
+import { KIBANA_LOCALIZATION_STATS_TYPE } from '../constants';
+
 export interface UsageStats {
   locale: string;
   integrities: Integrities;
   labelsCount?: number;
```

Comment:
Why is this file moving out of the `telemetry` plugin?

Reply:
because it depends on the i18n configuration. In NP, we won't have access to other plugins' config.
Also: the way usage collection works is for every plugin to report its own specific telemetry. So it kind of makes more sense to me for the `i18n` to report the telemetry from here rather than exposing the config so the telemetry plugin can report it. 
But happy to change it if necessary :)